### DISCUSSION TITLE
Latest release 0.9.2 not listed on website of project

### DISCUSSION THREAD

**Discussion by MR-DOS** [inventree/InvenTree#4280] [General]
Hi, release notes on website mention 0.9.1 as last version while 0.9.2 is available for quite some time. Not sure if I should file it as a regular bug.

**Comment by SchrodingersGat** [comment]
@MR-DOS can you post a link? 0.9.2 is listed in the following places:

- https://docs.inventree.org/en/latest/releases/release_notes/
- https://github.com/inventree/InvenTree/releases/tag/0.9.2

**Reply by MR-DOS** [reply]
If you ask Google, it lands on this page...
https://docs.inventree.org/en/stable/releases/release_notes/

**Reply by SchrodingersGat** [reply]
Thanks @MR-DOS this has now been fixed :)
